Paccy10/olympus-api
# Feature(categories): update category

**What does this PR do?**

- Update category

**Description of Task to be completed?**

- An authenticated admin user can update category

**How should this be manually tested?**

- Checkout to the branch `ft-update-category`
- Run the app with `flask run`
- Go to postman and send a PUT request to /categories/{name} with the required payload and replace `name` with the category name
- The named category should be updated

**Any background context you want to provide?**

- N/A

**What are the relevant pivotal tracker stories?**

- N/A

**Screenshots (if appropriate)**

- N/A

**Questions:**

- N/A
